src/General/Typography/Paragraph.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import {
+  Paragraph as StyledParagraph,
+  ParagraphProps,
+} from './ParagraphStyles';
+
+const Paragraph: React.FunctionComponent<Props> = props => {
+  const {
+    className,
+    children,
+    variant = 'regular',
+    bold,
+    color,
```

Comment:
Could move default color from `Styles.ts` file to here for putting all default props together?